StylableWinFormsControls/StylableWinFormsControls/Controls/StylableLabel.cs

```diff
@@ -87,7 +83,7 @@ internal virtual TextFormatFlags CreateTextFormatFlags(Size constrainingSize)
     /// Source: MeasureTextCache in System.Windows.Forms.Layout.LayoutUtils (part of it)
     /// Cache mechanism added for VSWhidbey 500516
     /// 3000 character strings take 9 seconds to load the form
-    internal class MeasureTextCache
+    public class MeasureTextCache

```

Comment:
Maybe we should extract this class to be in a separate File? This feature can be useful wherever we want to measure the width of a string

Reply:
and maybe rename it? It does not cache anything so the naming is a bit weird...

Reply:
I've extracted the internal classes into separate files now and used the InvalidateCache method from the MeasureTextCache, that one was simply not connected 😄